### ISSUE TITLE
[ORACLE] Error in CREATE VIEW DDL in synonym.sql, includes full name of view/table with schema same as in source

### ISSUE THREAD

**Issue by priyanshi-yb** [yugabyte/yb-voyager#673]
While exporting SYNONYMs from Oracle, it exports the `CREATE OR REPLACE VIEW` DDLs with full name of view with schema_name included and that schema will be same as the schema in which the synonym is present in Oracle sourceDB, but the schema with that name is not present in target and hence import schema fails with error of `does not exist`.

Example- 
```
CREATE OR REPLACE VIEW sakila_demo.offices AS SELECT * FROM sakila_demo.locations;
```
Error in import schema - 
```
ERROR: relation "sakila_demo.locations" does not exist (SQLSTATE 42P01)
```

as `locations` table is imported in public schema.

**Comment by priyanshi-yb** [comment]
There are these two options to decide between on one for the solutions - 

- Adding this case in analyze-schema and leaving it on user to do the needful manually like removing the schema name from the DDL or create the schema/required objects in the respective schema in the target.
- Voyager explicitly remove the schema name from the DDLs at the end of export-schema phase.

**Comment by priyanshi-yb** [comment]
fixed using option-2 in this PR #701
